prague-postgres-meetup-oct-2024.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Prague PostgreSQL Meetup on October 29th'
+description: 'The sixth Prague PostgreSQL Meetup of 2024 will take place on October 29th!'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/prague-postgres-meetup/prague-postgres-meetup@2x.png
+  alt: Prague PostgreSQL Meetup on October 29th
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 10-28-2024
+tags:
+  [
+    'postgres',
+    'postgresql',
+    'prague-postgres-meetup',
+    'prague',
+    'praha',
+    'meetup',
+    'event',
+    'community',
+    'october',
+    'pgconfeu'
+  ]
+published: true
+slug: prague-postgres-meetup-october-2024
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/prague-postgres-meetup/prague-postgres-meetup-og@2x.png
+---
+
+## Prague PostgreSQL Meetup on October 29th
+
+[Last week](https://xata.io/blog/pgconfeu-2024), I joined almost 800 other PostgreSQL enthusiasts at [PGConf.EU 2024](https://2024.pgconf.eu/) in Athens, Greece.
+Post-conference blues are real, but being back in Prague, I’m already looking forward to the Prague PostgreSQL Meetup on Tuesday, October 29th!
+
+![Andrew, Gulcin, and Rudy at PGConf.EU 2024](/images/prague-postgres-meetup/pgconfeu-2024.png)
+
+[Prague PostgreSQL Meetup](https://www.meetup.com/prague-postgresql-meetup/) is an active community of PostgreSQL enthusiasts in Prague, Czech Republic.
+I am happy to be the main organizer and greatly appreciate all the speakers, attendees, and sponsors who have made this meetup possible every month since 2018!
+
+![Prague PostgreSQL Meetup October](/images/prague-postgres-meetup/andrew-gulcin-rudy.png)
+
+I'm thrilled to invite you to our sixth meetup of 2024, taking place on Tuesday, October 29th, at 18:00.
```

Comment:
This conflicts a little with the previous sentence. Every month vs. six meetups in 2024.

Reply:
Ah, good catch. There are months we skip like summer months, December, Jan (because conference) etc. I will hide the every month phrase, thanks!